trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc

```diff
@@ -0,0 +1,50 @@
+Dessen Bedeutung wird oft nicht erkannt, aber die Pflege einer Organisation und ihrer Produkte ist tatsächlich entscheidend.
+
+Ein weiterer, offensichtlicher Grund für jemanden dabeizubleiben ist, dass er die Zusammenarbeit mit anderen Mitgliedern der Organisation genießen, einschließlich der _Trusted Committers_.
+Eine gedeihende Organisation ist eine, deren Mitglieder sich gegenseitig mit dem höchsten Respekt behandeln und miteinander kommunizieren.
+Beiträge werden mehr wie Geschenke oder Spenden denn als Ablenkungen behandelt, und sehr gute Beiträge (insbesondere erste) werden gelobt.
+Die Aufgabe des _Trusted Committers_ besteht in erster Linie darin, ein Vorbild für andere zu sein, analog zu einem Vorbild bezüglich der erwarteten Softwarequalität zu sein.
+Falls notwendig, sollten die _Trusted Committers_ einen Verhaltenskodex entwerfen und beschließen. Falls sich Mitglieder der Organisation entgegen den Regeln der Organisation oder gar schädigend verhalten,
+ist es die Aufgabe des _Trusted Committers_, dies anzusprechen. _Trusted Committers_ sollten Möglichkeiten für Zusammenkünfte der Mitglieder bieten, dafür sorgen, sich gegenseitig kennenzulernen und auftretende
+Konflikte einvernehmlich zu lösen.
+
+Menschen neigen auch dazu sich zu beteiligen, weil die Mitarbeit in einer _InnerSource Community_ eine hervorragende Gelegenheit bietet, neue Fähigkeiten zu erlernen und persönlich zu wachsen.
+Auch dies ist ein Aspekt, in dem die Rolle des _Trusted Committers_ wirklich wichtig ist.
+_Trusted Committers_ sind oft Mentoren für jüngere Entwickler, und verwenden ausdrücklich Zeit während der Pull Requests, um nicht nur auf Verbesserungspotentiale hinzuweisen, sondern auch im Detail zu erklären, warum und wie etwas
+verbessert werden muss. Sie liefern die Theorie oder das Hintergrundwissen zu Verbesserungen und schlagen die besten Möglichkeiten vor, Änderungen oder Verbesserungen zu implementieren.
+Dadurch können _Trusted Committers_ die Lerngeschwindigkeit ihrer Organisation weit über die traditioneller Software-Entwicklungsorganisationen erhöhen.
+
+Wir glauben, dass _Trusted Committers_ das Anwerben und Betreuen während der Pull Requests über das Erreichen von Veröffentlichungsterminen stellen sollten, außer es sprechen triftige Gründe dagegen.
+Gute Betreuung während eines Pull Requests führt zu höherem Vertrauen und Einbindung von https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_], was im Gegenzug zu mehr Mitarbeit führt.
+Wir werden das in https://innersourcecommons.org/resources/learningpath/trusted-committer/04/["Upleveling the Community"] näher betrachten.
+
+Schlussendlich gibt es Menschen, die sich in _InnerSource Communities_ engagieren, weil sie sich dort auf das Entwickeln von Software konzentrieren können, anstatt Aufgaben nachzugehen, die häufig als überflüssig oder unnötig angesehen werden.
```

Comment:
add "anderen" to change the sentence to  .... "anstatt anderen Aufgaben nachzugehen..."

Remove the "," before "anstatt"

Reply:
done

Reply:
done